### ISSUE TITLE
Embed signature into PDF on iOS

### ISSUE THREAD

**Issue by marcosatanaka** [mekentosj/podofo#8]
Hello,

I have a signature server to where I POST a hash (SHA-256) of the PDF, and receive the signature as a response. Now, I have to embed this signature on the PDF. I could not find this use case in the documentation. Is it possible to do this with PoDoFo?

**Comment by SunilAruruios** [comment]
Yes, this is  possible with podofo. For support you can contact me at sunil.aruru@gmail.com
